models/model-cards/gnn-fsi-modelcard.md

```diff
@@ -0,0 +1,183 @@
+
+### Name the adversely impacted groups (protected classes) this has been tested to deliver comparable outcomes regardless of:
+* Not Applicable
+
+### List the technical limitations of the model.
+* This model version requires a transactional data schema with entities (user, merchant, transaction) as requirement for the model.
+
+### What performance metrics were used to affirm the model's performance?
+* Area under ROC curve and Accuracy
+
+### What are the potential known risks to users and stakeholders? 
+* Not Applicable
+
+### What training is recommended for developers working with this model?  If none, please state "none."
+* none
+### Link the relevant end user license agreement 
+* [Apache 2.0](https://github.com/nv-morpheus/Morpheus/blob/branch-23.07/LICENSE)
+
+## Model Card ++ Saftey & Security Subcard
+
+### Link the location of the training dataset's repository (if able to share).
+* [training dataset](models/datasets/training-data/fraud-detection-training-data.csv)
+
+### Is the model used in an application with physical safety impact?
+* No
+
+### Describe physical safety impact (if present).
+* Not Applicable
+
+### Was model and dataset assessed for vulnerability for potential form of attack?
+* No
+### Name applications for the model.
+* Fraud detection in credit card transaction with the defined schema.
+### Name use case restrictions for the model.
+* Only tested for fraud detection in transactional data application
+### Has this been verified to have met prescribed quality standards?
+* No
+
+### Name target quality Key Performance Indicators (KPIs) for which this has been tested.  
+* Not Applicable
+### Technical robustness and model security validated?
+* Not Applicable
+### Is the model and dataset compliant with National Classification Management Society (NCMS)?
+* Not Applicable
+
+### Are there explicit model and dataset restrictions?
+* No
+
+### Are there access restrictions to systems, model, and data?
+* No
+### Is there a digital signature?
+* No
+
+## Model Card ++ Privacy Subcard
+
+
+### Generatable or reverse engineerable personally-identifiable information (PII)?
+
+* Neither
+
+### Was consent obtained for any PII used?
+* Not Applicable
+
+### Protected classes used to create this model? (The following were used in model the model's training:)
+
+* None of the Above
+
+
+### How often is dataset reviewed?
+* Other: Not Applicable
+
+### Is a mechanism in place to honor data
+* Yes
+### If PII collected for the development of this AI model, was it minimized to only what was required? 
+* Not applicable
+
+### Is data in dataset traceable?
+* No
+
+### Scanned for malware?
+* Not applicable
+### Are we able to identify and trace source of dataset?
+* Not applicable
```

Comment:
Do we know where the dataset comes from/how it is generated?
If so, let's say `Yes` here.